lib/database_cleaner/safeguard.rb

```diff
@@ -1,3 +1,5 @@
+require "uri"
```

Comment:
I added this because running `rspec spec/database_cleaner/safeguard_spec.rb` without this require would need to `NameError`.